rsconnect/metadata.py

```diff
@@ -380,6 +380,9 @@ def sha1(s):
     return base64.urlsafe_b64encode(m.digest()).decode("utf-8").rstrip("=")
 
 
+CURRENT_APP_STORE_VERSION = 1
```

Comment:
Are there other use cases for this to be a global variable? If not then this can be replace with just the class initialization.